cmake/CMakeLists.txt

```diff
@@ -580,30 +603,21 @@ if(BUILD_LIB)
 
```

Comment:
Please also add `info.h` and, depending on the build configuration, also `mpi.h` and `exceptions.h`:
```
  if(NOT BUILD_MPI)
      install(FILES ${LAMMPS_SOURCE_DIR}/STUBS/mpi.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/lammps)
  endif()
  if(LAMMPS_EXCEPTIONS)
      install(FILES ${LAMMPS_SOURCE_DIR}/exceptions.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/lammps)
  endif()
```

Reply:
Done.